mlir/test/rocmlir-driver/sanity_wmma.mlir

```diff
@@ -2,15 +2,15 @@
 // and LLVM IR.
 
 // fp16 tests.
-// RUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-opt
-// RUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu --verify-passes | rocmlir-opt
-// RUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,rocdl --verify-passes --arch=gfx1100 | rocmlir-opt
-// RUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,rocdl --verify-passes --arch=gfx1100 | rocmlir-translate -gpu-module-to-rocdlir | opt -passes='default<O3>,strip' -S | llc -mcpu=gfx1100
-// RUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,binary --verify-passes --arch=gfx1100 | rocmlir-opt
+// NORUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-opt
+// NORUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu --verify-passes | rocmlir-opt
+// NORUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,rocdl --verify-passes --arch=gfx1100 | rocmlir-opt
+// NORUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,rocdl --verify-passes --arch=gfx1100 | rocmlir-translate -gpu-module-to-rocdlir | opt -passes='default<O3>,strip' -S | llc -mcpu=gfx1100
+// NORUN: rocmlir-gen --arch gfx1100 -wmma=on -p -t f16 | rocmlir-driver -kernel-pipeline=gpu,binary --verify-passes --arch=gfx1100 | rocmlir-opt
```

Comment:
Why are these commented out?

Reply:
Yes, sorry, it was my attempt to debug some of those tests. Fixed